doc/examples/00_prepare_timeseries.ipynb

```diff
@@ -276,81 +276,312 @@
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "86d5aa86",
+   "metadata": {},
+   "source": [
+    "### Creating equidistant time series for fluxes\n",
+    "There are several methods in pandas to create equidistant series. A flux describes a measured or logged quantity in a period of time, ressulting in a pandas Series. In this series, each flux is asssigned a timesstamp in the index. In Pastas, we assume the index is at the end of the period that belongs to each measurement. This means that the precipitation of march 5 2022 gets the timestamp '2022-03-06 00:00:00' (which can be counter-intuitive, as the index is now a day later). Therefore, when using pandas resample methods, we add two parameters: closed='right' and label='right'."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "19f3be77",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "# plot the original series\n",
+    "series"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "90fffac6",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "series.resample(\"6H\", closed=\"right\", label=\"right\").mean()"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "871598bf",
+   "metadata": {},
+   "source": [
+    "The resample-method of pandas basically is a groupby-method. This creates problems when there is not a single measurement in each group. This is demponstrated by the NaN at 2000-01-01 18:00:00. Also when there are two values in a group, these are averaged, even though one of the values only counts for one hour, and the other for 6 hours. This is demonstrated by the value of 3.5 at 2000-01-02 00:00:00.\n",
+    "\n",
+    "Because of these problems there is a method called 'timestep_weighted_resample' in Pastas. This method assumes the index is at the end of the period that belongs to each measurement, just like the rest of Pastas. Using this assumption, the method can calculate a new series, using an overlapping period weighted average:"
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "id": "0bd45b5a",
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "new_index = pd.date_range(series.index[0], series.index[-1], freq=\"6H\")\n",
+    "ps.ts.timestep_weighted_resample(series, new_index)"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "id": "1d232423",
+   "metadata": {},
+   "source": [
+    "We see the value at 2000-01-02 00:00:00 is now 3.833333, which is 1/6th of 3.0 (the original value at 2000-01-01 19:00:00) and 5/6th of 4.0 (the original value at 2000-01-02 00:00:00). This methods sets a NaN for the value at 2000-01-01 00:00:00, as the length of the period cannot be determined."
```

Comment:
Don't understand this comment: 
_This methods sets a NaN for the value at 2000-01-01 00:00:00, as the length of the period cannot be determined._
Why would you not fill in 0, if that is the original value for timestamp 2000-01-01 00:00:00?

Reply:
Because the method used needs to know the length of each period, and for the first value, this period cannot be determined. To keep it simple, we do not add a check for this first measurement yet.

Reply:
clear